libr/core/cconfig.c

```diff
@@ -2955,6 +2955,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF ("asm.noisy", "true", "Show comments considered noisy but possibly useful");
 	SETPREF ("asm.offset", "true", "Show offsets at disassembly");
 	SETPREF ("scr.square", "true", "Use square pixels or not");
+	SETPREF ("scr.vi", "false", "Use vi mode");
```

Comment:
"Use vi mode for input prompt"

Reply:
What about using `scr.prompt.vi` instead? There is one for autocompletion widget for example...